templates/HelloBlazorHybrid/Abstractions/ICounterService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Stl.Fusion;
+
+namespace HelloBlazorHybrid.Abstractions
+{
+    public interface ICounterService
+    {
+        [ComputeMethod]
+        Task<int> Get(CancellationToken cancellationToken = default);
+        Task Increment(CancellationToken cancellationToken = default);
+    }
+}
```

Comment:
There is an option in Rider allowing to add CR / CRLF to the end of every file on save - please turn it on :)